guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -0,0 +1,49 @@
+[id="listing-using-satellite-ansible-modules_{context}"]
+== Listing the {Project} Ansible Modules
+
+You can view the installed {Project} Ansible modules by listing the content of the following directory:
+
+ifeval::["{build}" == "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
+----
+
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs in the Automation Hub.
+
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+----
+
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://galaxy.ansible.com/theforeman/foreman in the Ansible Galaxy.
+
+endif::[]
+
+
+[NOTE]
+====
+At the time of writing, the `ansible-doc -l` command does not list collections yet.
+====
+
+ifeval::["{build}" == "satellite"]
+
+All modules are in the `redhat.satellite` namespace and can be referred to as follows:
+----
+$ ansible-doc redhat.satellite._module_name_
+----
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+
+All modules are in the `theforeman.foreman` namespace and can be referred to as follows:
+----
+$ ansible-doc theforeman.foreman._module_name
+----
+endif::[]
+
+
+Note that all modules require the `server_url`, `username`, and `password` parameters.
```

Comment:
This bit is explained in the documentation for each separate module. This sentence might become untrue in future, which makes it hard to maintain. I expect Ansible users to be able to find the required parameters for modules they use. I suggest removing this sentennce completely.

Reply:
Done